time stamp
int32 imageID
detector/BBPose[] object_poses